app/views/step_by_step_pages/schedule.html.erb

```diff
@@ -60,10 +60,28 @@
         ]
       } %>
 
-      <div class="form-group">
-        <%= label_tag "schedule[time]", 'Time' %>
-        <%= text_field_tag "schedule[time]" %>
-      </div>
+      <% time_errors = @step_by_step_page.issues.include?(:schedule_time) ? [
+        {
+          text: @step_by_step_page.issues[:schedule_time],
+        }
+      ] : false %>
+      <%= render "components/autocomplete", {
+        id: "time",
+        name: "schedule[time]",
+        label: {
+          text: t("schedule.edit.time"),
+          bold: true,
+        },
+        error_items: time_errors,
```

Comment:
I'm happy to make a change to the autocomplete component so that it can take an `error_message` like date_input does. But also quite keen to avoid making modifications to anything copied over from Content Publisher, so that we don't end up with two forks that grow further apart.